### ISSUE TITLE
Can't Open Avocode doc

### ISSUE THREAD

**Issue by ryeskelton** [avocode/avocode#329]
Describe your problem here…

-----

```
RangeError: Maximum call stack size exceeded
  at /Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/space-pen/lib/space-pen.js:259:9
  at Builder.openTag (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/space-pen/lib/space-pen.js:265:9)
  at Builder.tag (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/space-pen/lib/space-pen.js:239:12)
  at Function.View.(anonymous function) (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/space-pen/lib/space-pen.js:56:49)
  at Function.module.exports.e.content (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/avocode-view.js:1:781)
  at Function.<anonymous> (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/space-pen/lib/space-pen.js:122:31)
  at Function.View.buildHtml (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/space-pen/lib/space-pen.js:96:10)
  at e.View (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/space-pen/lib/space-pen.js:121:33)
  at new e (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/avocode-view.js:1:632)
  at t._addRevisionView (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:13966)
  at t._openAvocodeResource (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:13327)
  at t.opener (/Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:11769)
  at /Users/Rye/Downloads/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:950
  at wrappedOpener (/Users/Rye/Downloads
```

**Comment by petrbrzek** [comment]
Can you describe us how did it happen?

**Comment by rychlis** [comment]
I'm closing this issue as there was no response. We've also released Avocode 2.3.0 today, please re-report the issue if you can reproduce the problem in 2.3.0
